src/components/common/RowSection/RowSection.js

```diff
@@ -11,7 +13,19 @@ const RowSection = ({ title, align, books, ...props }) => {
       </Typography>
       <div className="RowSection__row">
         {books?.map((book) => (
-          <BookCard key={book._id} book={book} />
+          <BookCard
+            key={book._id}
+            book={book}
+            cardHeight="auto"
+            imageProps={{ width: '100%', height: '18rem' }}
+            buttonProps={{
+              onClick: (e) => {
```

Comment:
Take this onclick method out side into a function

Reply:
The inline function is part of button props object. Hence, this way.